_site/acknowledgments.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
```

Comment:
Configure files in the `_site` directory to be ignored by git using a `.gitignore` file (such as in [this example](https://github.com/github/gitignore/blob/master/Jekyll.gitignore)). That's because these files are generated by jekyll itself.